prime-router/src/main/kotlin/Mappers.kt

```diff
@@ -520,6 +520,7 @@ class Obx8Mapper : Mapper {
                 "840535000" -> "A" // Antibody to severe acute respiratory syndrome coronavirus 2 (substance)
                 "840534001" -> "A" // Severe acute respiratory syndrome coronavirus 2 vaccination (procedure)
                 "373121007" -> "N" // Test not done
+                "82334004" -> "N" // Indeterminate
```

Comment:
This obx8 mapper seems like it should be in a value set.   Or maybe its a good first candidate for @carlosfelix2 's database table lookup feature.